shared/package.json

```diff
@@ -7,7 +7,7 @@
         "_node": "cross-env-shell BABEL_ENV=yarn babel-node --extensions='.tsx' --extensions='.js' ",
         "help": "yarn run _helper help",
         "lint": "yarn run lint-specific .",
-        "lint-specific": "eslint --cache --quiet --ext js,ts,tsx,d.ts,mobile.tsx,desktop.tsx",
```

Comment:
intentionally kept?

Reply:
the new ts toolchain can't analyze js files ut we only have like 3 of them and they never change and they're small